pages/index.tsx

```diff
@@ -1,71 +1,13 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import React from 'react'
+import Homepage from '../src/screens/Homepage';
+import Fullscreen from '../src/layout/Fullscreen';
 
-export default function Home() {
+const index = () => {
   return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
```

Comment:
Why we removed `Head`? 

Head component is useful to generate metadata for the page 

Please have a look
https://nextjs.org/docs/api-reference/next/head